fv_be/app/admin/admin.py

```diff
@@ -5,11 +5,30 @@
 from django.contrib.sites.models import Site as Sites
 from rest_framework.authtoken.models import TokenProxy
 
+from fv_be.app.models.category import Category
+from fv_be.app.models.dictionary import (
+    Acknowledgment,
+    AlternateSpelling,
+    DictionaryEntry,
+    Note,
+    Pronunciation,
+    Translation,
+)
+from fv_be.app.models.part_of_speech import PartOfSpeech
 from fv_be.app.models.sites import Site
 
 from .base_admin import BaseAdmin
 from .sites_admin import MembershipInline
 
+dictionary_models = [
```

Comment:
There is no specific purpose, I was just trying to write more concise code, i.e. just registering this list instead of registering 6 models separately coming from the same file. I believe it may be similar in your case of character models as well.